apps/ai-content-generator/src/components/app/dialog/common-generator/CommonGenerator.tsx

```diff
@@ -41,11 +41,13 @@ const CommonGenerator = (props: FeatureComponentProps) => {
   useEffect(updateProviderData, [parameters.targetLocale, parameters.locale, dispatch]);
 
   return (
-    <Flex>
-      <NewOrExistingText isNewText={parameters.isNewText} dispatch={dispatch} />
-      <FieldSelector parameters={parameters} isTranslate={isTranslate} fieldTypes={TextFields} />
+    <div className={styles.root}>
```

Comment:
Nonblocking Q: Do we want to have these styles be applied inside of the components their meant for instead of at the top level?